src/server/index.d.ts

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+export interface kbn_server {
+    config: () => configObject;
+  }
+  
+export interface configObject {
+    get: (path: string) => ConfigEntry;
+}
+
+export interface ConfigEntry {
+    zoom: number;
+    viewport: ViewWidthHeight;
+  }
+  
+export interface ViewWidthHeight {
```

Comment:
We already have a type for this in reporting code, it's just buried.  https://github.com/elastic/kibana/blob/master/x-pack/plugins/reporting/server/browsers/chromium/driver/screenshot_stitcher/types.ts#L14

Can move that file to the top of the reporting plugin, and maybe rename is `types.d.ts` since there is no code to execute in it.  Then can use `Size` instead of `ViewWidthHeight`.